notebooks/en/function_calling_fine_tuning_llms_on_xlam.ipynb

```diff
@@ -0,0 +1,14340 @@
+{
```

Comment:
<p>Let's remove the subsections here as they interfere with the correct rendering of the table of contents. Instead, we could use bold text.</p><p>Additionally, I'd remove the need to use &lt;small&gt;</p><p>Have we tested it on Colab? Remember all the recipes in Cookbook include an Open in Colab button, so it's important to consider that platform and add details about it if needed :)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>